### ISSUE TITLE
Ruffle Error on https://www.breadalbanestudy.club/game.php?ID=191

### ISSUE THREAD

**Issue by seannorris** [ruffle-rs/ruffle#2109]
# Error Info
Error name: Error
Error message: panicked at 'called `Option::unwrap()` on a `None` value', core/src/display_object/container.rs:780:18
Error stack:
```
X@https://sn76.host.cs.st-andrews.ac.uk/ruffle/core.ruffle.0209bbc2d84b1cf4bfff.js:1:20832
__wbg_new_df6e6ab7a65c4c4d@https://sn76.host.cs.st-andrews.ac.uk/ruffle/ruffle.js:1:37870
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[1310]:0x252826
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[1476]:0x270226
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[4019]:0x333031
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[3135]:0x31134b
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[4328]:0x338013
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[4358]:0x338576
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[4230]:0x336b2a
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[4359]:0x3385ac
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[4148]:0x3355b7
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[670]:0x1a110a
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[1075]:0x2204d8
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[338]:0xc14d7
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[903]:0x1f30b9
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[1491]:0x2728b1
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[973]:0x206233
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[279]:0xc876
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[338]:0xc2297
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[903]:0x1f30b9
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[279]:0x1f400
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[338]:0xc2297
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[903]:0x1f30b9
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[839]:0x1df5c7
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[416]:0x112594
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[957]:0x201b95
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[3495]:0x3239e0
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[819]:0x1d8a02
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[4258]:0x337140
@https://sn76.host.cs.st-andrews.ac.uk/ruffle/3879385cefe79eb75909.module.wasm:wasm-function[4529]:0x33a272
P@https://sn76.host.cs.st-andrews.ac.uk/ruffle/core.ruffle.0209bbc2d84b1cf4bfff.js:1:18605
f@https://sn76.host.cs.st-andrews.ac.uk/ruffle/core.ruffle.0209bbc2d84b1cf4bfff.js:1:17715

```

# Player Info
Allows script access: false

# Page Info
Page URL: https://www.breadalbanestudy.club/game.php?ID=191
SWF URL: https://www.breadalbanestudy.club/SWFs/TheBindingOfIsaacWrathOfTheLamb.swf

# Browser Info
Useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:84.0) Gecko/20100101 Firefox/84.0
OS: Win32

# Ruffle Info
Version: 0.1.0
Name: nightly 2020-12-26
Channel: nightly
Built: 2020-12-26T00:39:49.720Z
Commit: f74fc3abe3eb0c20badf8479d0e8f811447f0734

**Comment by Poopooracoocoo** [comment]
#1991

**Comment by Toad06** [comment]
Thanks for the report.

This is an issue that seems to imply the value of `_parent`, just like #1991 and a few other ones. Compiling with the changes mentioned in https://github.com/ruffle-rs/ruffle/issues/1991#issuecomment-746398537 prevents the crash from happening.

**Comment by Toad06** [comment]
Fixed by #2228.
